### PR TITLE
feat(Notebooks) : add notebooks page

### PR DESCRIPTION
**Pull request by cheikhgwane**

### CODE REVIEW

File: src/notebooks/graphql/queries.graphql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+query notebooksUrl{
```
Comment (qgerome) [inline src/notebooks/graphql/queries.graphql:1]: Wouldn't that be more NotebooksPage query ?

File: src/pages/notebooks/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  );
+};
+
+export const getServerSideProps = createGetServerSideProps({
+  requireAuth: true,
+  getServerSideProps: async (ctx, client) => {
+    await client.query({
+      query: NotebooksUrlDocument,
+    });
+    return { props: {} };
```
Comment (qgerome) [inline src/pages/notebooks/index.tsx:27]: Not needed

File: src/pages/notebooks/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { createGetServerSideProps } from "core/helpers/page";
+import {
+  NotebooksUrlDocument,
+  useNotebooksUrlQuery,
+} from "notebooks/graphql/queries.generated";
+
+const NoteBooksPage = () => {
```
Comment (qgerome) [inline src/pages/notebooks/index.tsx:7]:
```suggestion
const NotebooksPage = () => {
```

File: src/notebooks/graphql/queries.graphql (hunk 1 above)
Comment (cheikhgwane) [reply src/notebooks/graphql/queries.graphql:1]: Yes, will update it

File: src/pages/notebooks/index.tsx (hunk 2 above)
Comment (cheikhgwane) [reply src/pages/notebooks/index.tsx:27]: Okay

File: src/notebooks/graphql/queries.graphql [hunk 4]
```diff
@@ -1,3 +1,3 @@
-query notebooksUrl{
+query noteBooksPage{
```
Comment (qgerome) [inline src/notebooks/graphql/queries.graphql:1]:
```suggestion
query notebooksPage{
```

Comment (qgerome) [reply src/notebooks/graphql/queries.graphql:1]: You kept the spelling in two words

**Review by qgerome — CHANGES_REQUESTED** [review]

File: src/notebooks/graphql/queries.graphql (hunk 4 above)
Comment (cheikhgwane) [reply src/notebooks/graphql/queries.graphql:1]: Oh I kept the old one. Will update it

**Pull request merged**